src/Lykke.Job.BlockchainCashoutProcessor/Modules/BlockchainsModule.cs

```diff
@@ -27,9 +33,30 @@ protected override void Load(ContainerBuilder builder)
                 .WithParameter(TypedParameter.From(_blockchainWalletsServiceClientSettings.ServiceUrl))
                 .SingleInstance();
 
-            builder.RegisterType<HotWalletsProvider>()
-                .As<IHotWalletsProvider>()
-                .WithParameter(TypedParameter.From<IReadOnlyDictionary<string, string>>(_blockchainsIntegrationSettings.Blockchains.ToDictionary(b => b.Type, b => b.HotWalletAddress)))
+            var blockchainConfigurations = new Dictionary<string, BlockchainConfiguration>();
+
+            foreach (var blockchain in _blockchainsIntegrationSettings.Blockchains.Where(b => !b.IsDisabled))
+            {
+                _log.WriteInfo("Blockchains registration", "", 
+                    $"Registering blockchain: {blockchain.Type} -> \r\nHW: {blockchain.HotWalletAddress}");
+
+                if (blockchain.AreCashoutsDisabled)
+                {
+                    _log.WriteWarning("Blockchains registration", "", $"Cashouts for blockchain {blockchain.Type} are disabled");
+                }
+
+                var blockchainConfiguration = new BlockchainConfiguration
+                (
+                    blockchain.HotWalletAddress,
+                    blockchain.AreCashoutsDisabled
+                );
+
+                blockchainConfigurations.Add(blockchain.Type, blockchainConfiguration);
+            }
+
+            builder.RegisterType<BlockchainConfigurationsProvider>()
```

Comment:
```blockchainConfigurations``` could be empty. This will not lead to any unexpected errors. Any cashout will lead retries of the to ```StartCashoutCommand```, just like with not empty ```blockchainConfigurations``` but with cashout for not configured yet blockchain.